domutil.go

```diff
@@ -1,12 +1,24 @@
 package vecty
 
+func parentNode(node jsObject) jsObject {
+	if node == nil {
+		panic("vecty: Cannot find parent of nil node")
+	}
+	parent := node.Get("parentNode")
+	if parent == nil {
+		panic("vecty: Missing parent node")
+	}
```

Comment:
Okay. I'd prefer to drop this part of the change if you're alright with that. I think that a package should really only panic (esp. with a `vecty:` prefix) if the user of the library did something obviously wrong. In this case.. I don't think this will be triggered due to a user error, so I'd prefer to keep the (less informative) nil pointer panic here.